packages/radix/src/components/GhostButton.tsx

```diff
@@ -28,6 +28,7 @@ export const GhostButton = React.forwardRef<HTMLButtonElement, GhostButtonProps>
               lineHeight: theme.lineHeights[2],
               height: theme.sizes[5],
               minWidth: theme.sizes[5],
+              mixBlendMode: 'multiply',
```

Comment:
![I've got the power GIF](https://user-images.githubusercontent.com/8441036/76879844-7e75b500-687f-11ea-8bea-97fcc9d0b25e.gif)

Reply:
lol can we do that in Modulz?

Reply:
I think we should, shouldn't we?

Reply:
That’s probably the only setting designers will know lol